.eslintrc.js

```diff
@@ -4,13 +4,13 @@ module.exports = {
     parser: 'babel-eslint',
     sourceType: 'module'
   },
-  extends: [
-    '@nuxtjs'
-  ],
+  extends: ['@nuxtjs'],
   rules: {
     'vue/singleline-html-element-content-newline': 0,
     'vue/multiline-html-element-content-newline': 0,
     'vue/html-self-closing': 0,
-    'vue/no-v-html': 0
+    'vue/no-v-html': 0,
+    'arrow-parens': 0,
+    semi: 0
```

Comment:
According comment from @nielsdB97 ,

> ... Also I think the formatting changes should be separate from this PR. Now it's a bit hard to see the actual changes.

can we revert this configuration first back to the original. It will be easier for reviewer to see the actual changes. And also the one in `.editorconfig` as well.